.plotrc

```diff
@@ -49,6 +49,7 @@ if [[ -d "${__SCIROOPLOT_BUILD_DIR}" ]]; then
     mkdir ${PROJECT_NAME}/build
     cp ${__SCIROOPLOT_DIR}/skeleton/* ${PROJECT_NAME}/
     sed -i.bak "s/MyPlots/${PROJECT_NAME}/" ${PROJECT_NAME}/CMakeLists.txt && rm ${PROJECT_NAME}/CMakeLists.txt.bak
+    sed -i -e "s/\"myPlots\"/\"${PROJECT_NAME}\"/g" ${PROJECT_PATH}/CreatePlots.cxx
```

Comment:
here you will also need the .bak magic as in the line above, because otherwise it will not work on mac

Reply:
does the option .bak create a backup file with ending .bak or what is happening here?

Reply:
yes.
this is the only portable solution because sed works slightly different on mac and linux